dockstore-integration-testing/src/test/java/io/dockstore/client/cli/NotificationIT.java

```diff
@@ -0,0 +1,133 @@
+package io.dockstore.client.cli;
+
+import java.sql.Timestamp;
+import java.util.List;
+
+import io.dockstore.common.ConfidentialTest;
+import io.swagger.client.ApiClient;
+import io.swagger.client.ApiException;
+import io.swagger.client.api.CurationApi;
+import io.swagger.client.model.Notification;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertEquals;
+
+@Category(ConfidentialTest.class)
+public class NotificationIT extends BaseIT {
+
+    // set up test apis as it would be for an admin and a regular user
+    private final ApiClient webClientAdmin = getAdminWebClient();
+    private CurationApi curationApiAdmin = new CurationApi(webClientAdmin);
+
+    private final ApiClient webClientUser = getWebClient(USER_1_USERNAME, testingPostgres);
+    private CurationApi curationApiUser = new CurationApi(webClientUser);
+
+    private Notification testNotification() {
+        Notification notification = new Notification();
+        notification.setMessage("holla");
+        notification.setExpiration(new Timestamp(100000));  // a past timestamp
+        notification.setPriority(Notification.PriorityEnum.HIGH);
+        return notification;
+    }
+
+    private Notification anotherTestNotification() {
+        Notification notification = new Notification();
+        notification.setMessage("ayy");
+        notification.setExpiration(new Timestamp(System.currentTimeMillis() + 100000));  // a future timestamp
+        notification.setPriority(Notification.PriorityEnum.HIGH);
+        return notification;
+    }
+
+    @Test
+    public void testGetNotifications() {
+
+        // enter test notifications in the database
+        Notification expired = testNotification();
+        Notification current = anotherTestNotification();
+        curationApiAdmin.createNotification(expired);
+        curationApiAdmin.createNotification(current);
+
+        // get notifications and confirm that only the non-expired one is returned
+        List<Notification> activeNotifications = curationApiUser.getActiveNotifications();
+        assertEquals(activeNotifications.size(), 1);
+        assertEquals(activeNotifications.get(0).getMessage(), "ayy");
```

Comment:
You could have a variable/constant for "ayy" so that you don't have the same string twice. 

Suggest using IDE's refactoring feature to implement this. :)